AccessibilityInsightsForAndroidService/app/src/main/java/com/microsoft/accessibilityinsightsforandroidservice/ResultRequestFulfiller.java

```diff
@@ -68,7 +74,9 @@ private String getScanContent(AccessibilityNodeInfo rootNode, Bitmap screenshot)
       throw new ScanException("Scanner returned no data");
     }
 
-    resultSerializer.addAxeResult(result);
-    return resultSerializer.generateResultJson();
+    List<AccessibilityHierarchyCheckResult> results =
+        atfaScanner.scanWithATFA(rootNode, new BitmapImage(screenshot));
+
+    return resultSerializer.createResultsJson(result, results);
```

Comment:
It was a little confusing what result and results were at first glance. Perhaps it could be less confusing if we rename them to something like axeResult and atfaResults?

Reply:
Good call! In the spirit of name improvement, I also updated `ResultSerializer` to `ResultsContainerSerializer`